src/features/canvass/components/LocationDialog/index.tsx

```diff
@@ -36,7 +37,8 @@ type LocationDialogStep =
   | 'household'
   | 'editHousehold'
   | 'locationVisit'
-  | 'householdVisit';
+  | 'householdVisit'
+  | 'householdsVisit';
```

Comment:
One idea would be to use something slightly more different for this to avoid spelling (and reading) mistakes, e.g. `bulkHouseholdVisits`.

I think this is also more consistent with other parts of the code, where "household visits" is pluralized, instead of pluralizing household as "households visit" (which is currently the case here, but not in the store or the hook).